test/src/sdk-harness/test_projects/tx_fields/mod.rs

```diff
@@ -142,6 +147,41 @@ async fn add_message_input(tx: &mut ScriptTransaction, wallet: WalletUnlocked) {
     tx.tx.inputs_mut().push(message_input);
 }
 
+async fn get_output_predicate() -> (WalletUnlocked, Predicate, AssetId) {
```

Comment:
The convention in Rust is to not prepend names with `get_`

Reply:
This follows the same syntax as other functions with similar functionality in this file.